packages/mongo-select/src/MongoSelect.tsx

```diff
@@ -0,0 +1,228 @@
+  color: ${uiColors.blue.base};
+`;
+
+const projectButtonStyle = css`
+  display: flex;
+  justify-content: space-around;
+  padding-top: 16px;
+  padding-bottom: 16px;
+`;
+
+const Variant = {
+  Organization: 'organization',
+  Project: 'project',
+} as const;
+
+type Variant = typeof Variant[keyof typeof Variant];
+
+export { Variant };
+
+const PlanType = {
+  Cloud: 'cloud',
+  Atlas: 'Atlas',
+  OM: 'OM',
+} as const;
+
+type PlanType = typeof PlanType[keyof typeof PlanType];
+
+export { PlanType };
+
+export interface ProjectInterface {
+  projectId: string;
+  projectName: string;
+  planType: PlanType;
+  orgId: string;
+}
+
+export interface OrganizationInterface {
+  orgId: string;
+  orgName: string;
+  planType: PlanType;
+}
+
+function isProject(
+  val: ProjectInterface | OrganizationInterface,
+): val is ProjectInterface {
+  return 'projectId' in val;
+}
+
+const onKeyDown: React.KeyboardEventHandler = e => {
+  if ([keyMap.ArrowUp, keyMap.ArrowDown].includes(e.keyCode)) {
+    e.preventDefault();
+  }
+};
+
+type VariantData = ProjectInterface | OrganizationInterface;
+
+interface MongoSelectProps {
+  /**
+   * Object with information about current organization or project.
+   * Organization: {orgId: `string`; orgName: `string`, planType: `'Cloud' | 'OM' | 'Atlas'`}
+   * Project: {orgId: `string`; projectId: `string`, projectName: `string`, planType: `'Cloud' | 'OM' | 'Atlas'`}
+   */
+  selected: VariantData;
+
+  /**
+   * Array of data objects
+   * Organization: [{orgId: `string`; orgName: `string`, planType: `'Cloud' | 'OM' | 'Atlas'`}]
+   * Project: [{orgId: `string`; projectId: `string`, projectName: `string`, planType: `'Cloud' | 'OM' | 'Atlas'`}]
+   */
+  data: Array<VariantData>;
+
+  /**
+   * Callback function executed when an organization is clicked.
+   */
+  onClick?: React.MouseEventHandler;
+
+  /**
+   * Determines variant 'organization' | 'project'
+   */
+  variant: Variant;
+}
+
+/**
+ * # MongoSelect
+ *
+ * MongoSelect component
+ *
+ * ```
+<MongoSelect onClick={onClick} selected={'YouWork'} data={[{name: 'YouWork', product: 'Atlas'}]/>
+```
+ * @param props.selected Selected organization, which will appear in the Top Nav by default.
+ * @param props.data Array of Organization objects, [{name: `string`, product: `string`}].
+ * @param props.onClick Callback function executed when an organization is clicked.
+ * @param props.variant Determines if MongoSelect will have `organization` or `project` data
+ */
+function MongoSelect({ selected, data, onClick, variant }: MongoSelectProps) {
+  const [open, setOpen] = useState(false);
+  const [filteredData, setFilteredData] = useState(data);
+
+  const onChange: React.ChangeEventHandler = ({ target }) => {
+    const fuse = new Fuse(data, { keys: ['orgName', 'projectName'] });
+    const results = fuse.search((target as HTMLInputElement).value);
+
+    setFilteredData(results);
+  };
+
+  let trigger, footer;
+  const orgURI = `/v2#/org/${selected.orgId}`;
```

Comment:
Ditto these links..need to figure out how to handle non-cloud sub-domains so users are routed correctly.